pkg/common/config/config.go

```diff
@@ -76,6 +76,8 @@ const (
 	// interval after which stale CnsVSphereVolumeMigration CRs will be cleaned up.
 	// Current default value is set to 24 hours.
 	DefaultCnsVolumeOperationRequestCleanupIntervalInMin = 1440
+	// DefaultMaxBlockVolumeSnapshotsPerVolume is the default maximum number of block volume snapshots per volume.
+	DefaultMaxBlockVolumeSnapshotsPerVolume = 3
```

Comment:
Per the discussion in design review meeting, we agreed to name the parameter to be explicitly distinguishable between the global one and the future granular ones.

For example, `global-maximum-per-block-volume` and `granular-maximum-per-block-volume-vsan` / `granular-maximum-per-block-volume-vvol` are what we have in the Design doc right now.

Reply:
I did see that, but I don't think it immediately conveys what configuration it is. How about `global-max-block-volume-snapshots-per-volume` ? This way we are distinguishing it from the file volume snaps, while also clearly indicating what configuration it is.

Reply:
Sounds ok. Or, `global-max-snapshots-per-block-volume`, which would be simpler and also self-explanatory.

Reply:
wanted to differentiate between file and block volume, just like our FSS

Reply:
`per-block-volume` in `global-max-snapshots-per-block-volume` clear indicates and differentiates between file and block volumes, doesn't it?

Reply:
fair enough. Will change.

Reply:
Done.

Reply:
Do we need to repeat "volume"?  How about "DefaultMaxSnapshotsPerBlockVolume"?